src/NUnitFramework/tests/Internal/RandomizerBiasTests.cs

```diff
@@ -58,18 +108,17 @@ public static void NextDecimalWithMaximumIsNotBiased()
             const decimal wholeRange = decimal.MaxValue * (2 / 3m);
             const decimal halfRange = wholeRange / 2;
 
-            const int totalCount = 10000;
```

Comment:
@rprouse Now that we require VS2017+, is there any reason not to remove the <LangVersion> restriction so that we can beginning using C# 7? Specifically here, I wanted to use `10_000` and that would have helped me and maybe reviewers of the first PR to spot the inconsistency with the other method's `100_000`.

Reply:
I think we should keep the `<LangVersion>` to make sure language features that aren't supported in VS 2017 don't creep in, but I'm fine with updating it to 7 from 6. I'm not sure, but I think we may require 15.3+ which would allow 7.1, but our `BUILDING.md` just says VS 2017, so 7 is safest.